modules/styles/product/src/main/resources/web/styles/css/main.css

```diff
@@ -12,6 +12,7 @@ table#main-table td#menu-panel {
 	border-right: solid 1px #989898;
 	background-position: right top;
 	background-repeat: repeat-y;
+	width: 230px;

```

Comment:
This is a global change that affects other places right?
IMO, this should be `min-width`.

Reply:
Yes, It affects the whole menu panel in IS Management console. Fixed with 2b9bcbd09c323e9049e1cf026cc132137dc3b907.